From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sami Tolvanen <samitolvanen@google.com>
Date: Thu, 13 Jun 2019 12:06:48 -0700
Subject: ANDROID: soc/tegra: disable ARCH_TEGRA_210_SOC with LTO

Disable CONFIG_ARCH_TEGRA_210_SOC with LTO to work around an issue
with ThinLTO.

Bug: 145210207
Change-Id: Ic37929da7337317ff2720f1f939227b99f0cdadd
Link: https://github.com/ClangBuiltLinux/linux/issues/510
Signed-off-by: Sami Tolvanen <samitolvanen@google.com>
---
 drivers/soc/tegra/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/tegra/Kconfig b/drivers/soc/tegra/Kconfig
index 84bd615c4a92..1213fe6432b1 100644
--- a/drivers/soc/tegra/Kconfig
+++ b/drivers/soc/tegra/Kconfig
@@ -80,6 +80,7 @@ config ARCH_TEGRA_210_SOC
 	select SOC_TEGRA_FLOWCTRL
 	select SOC_TEGRA_PMC
 	select TEGRA_TIMER
+	depends on !LTO_CLANG
 	help
 	  Enable support for the NVIDIA Tegra210 SoC. Also known as Tegra X1,
 	  the Tegra210 has four Cortex-A57 cores paired with four Cortex-A53
